assets/data/aspects/monsters.json

```diff
@@ -0,0 +1,8 @@
+[
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauven_guerilla",
+	"effects": [ "monsterDrauvenShred", "hunterSilkstep" ],
+	"boostType": "ABILITY_MISC"
```

Comment:
Note: `monsters.json` would be updating the enemy version of the skill, not the player version of the skill. We probably don't need to include this definition here.

Reply:
ability has been renamed to drauven teamwork to prevent this